File: problem1.rb
nat_num = []

i = 0

while i < 1000 do
  num = i
  if (num % 3 == 0) || (num % 5 == 0)
    nat_num << num
  end
  i+=1
end
final_num = 0 

for numbers in nat_num do
  final_num += numbers
end

puts final_num

File: pr1/problem1.js

var natNum = [];

var maxNum = prompt('Enter a max value');

var firstMultiple = prompt('Enter the first multiple');
var secondMultiple = prompt('Enter the second multiple');

for ( var i = 0; i < maxNum; i++){
  if ((i % 3 === 0) || (i % 5 === 0)){
    natNum.push(i);
  }
}

var finalNum = 0;
for ( var i = 0; i < natNum.length; i++){
  finalNum = finalNum + natNum[i];
}

console.log(finalNum);